lib/Sensors/sensors_state.dart

```diff
@@ -0,0 +1,29 @@
+class SensorState {
+  final double voltage;
+  final double power;
+  final String mqttVoltage;
+
+  const SensorState({
```

Comment:
it`s better to use Equatable

@OverRide
List get props => [voltage, power, mqttVoltage];